docs/src/content/docs/changelog.md

```
@@ -0,0 +1,384 @@
+### Fixed
+
+- Module issues
+
+## v3.0.0-alpha.5 - 2024-07-30
+
+### Added
+
+- [linux] WindowDidMove / WindowDidResize events in
+  [#3580](https://github.com/wailsapp/wails/pull/3580)
+- [windows] WindowDidResize event in
+  (https://github.com/wailsapp/wails/pull/3580)
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)